src/tools/annotation/bidirectionalTool/moveHandle/moveLongLine/moveLongLine.js

```diff
@@ -0,0 +1,60 @@
+import getDistanceWithPixelSpacing from '../../utils/getDistanceWithPixelSpacing.js';
+import getBaseData from '../getBaseData.js';
+import updatePerpendicularLine from './updatePerpendicularLine.js';
+
+/**
+ * Move the long line updating the perpendicular line handles position.
+ *
+ * @param {*} proposedPoint Point that was moved in bidirectional tool
+ * @param {*} measurementData Data from current bidirectional tool measurement
+ * @param {*} eventData Data object associated with the event
+ * @param {*} fixedPoint Point that is not being moved in long line
+ *
+ * @returns {boolean} True if perpendicular handles were updated, false if not
+ */
+export default function moveLongLine(
+  proposedPoint,
+  measurementData,
+  eventData,
+  fixedPoint
+) {
+  const baseData = getBaseData(measurementData, eventData, fixedPoint);
+  const {
+    columnPixelSpacing,
+    rowPixelSpacing,
+    distanceToFixed,
+    perpendicularStart,
+    perpendicularEnd,
+  } = baseData;
```

Comment:
You can one-line this (but don't have to):

```js
  const {
    columnPixelSpacing,
    rowPixelSpacing,
    distanceToFixed,
    perpendicularStart,
    perpendicularEnd,
  } = getBaseData(measurementData, eventData, fixedPoint);
```

Reply:
Actually I can but it's not a good idea.
If I do so, I'll be creating lots of warnings due to the ESLint restriction of 80 chars per line.
But those line breaks are also causing a lot of LoC issues on CodeClimate: https://codeclimate.com/github/cornerstonejs/cornerstoneTools/pull/1063#issues

Reply:
The `CodeClimate` restriction is arbitrary (afaik). We can bump it or ignore it.
A lot of static analysis is "hey, this is a little odd and could be a red flag, can you take a quick look?" Conceptually, I think having explicit knowledge of the variables your function requires is a cognitive victory.

Certainly open for debate -- happy to hear your thoughts.